src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -1005,4 +1005,20 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
     <value>This package version is deprecated. Use {0} instead.</value>
     <comment>{0} is a NuGet package id (string)</comment>
   </data>
+  <data name="Billion" xml:space="preserve">
+    <value>B</value>
+    <comment>1.000.000.000 in short format notation</comment>
+  </data>
+  <data name="Million" xml:space="preserve">
+    <value>M</value>
+    <comment>1.000.000 in short format notation</comment>
+  </data>
+  <data name="Thousand" xml:space="preserve">
+    <value>K</value>
+    <comment>1.000 in short format notation</comment>
```

Comment:
English uses `,` for thousands separators, so while all of these comments use `.` which is probably more common across the world, given our code and comments are in English, it looks a but odd, and this one (thousands) can be mistaken for the number one (with 4 digits of precision).